cpp/example_code/general/CMakeLists-minimal.txt

```diff
@@ -1,18 +1,26 @@
-# minimal CMakeLists.txt for the AWS SDK for C++
+# Minimal CMakeLists.txt for the AWS SDK for C++.
 cmake_minimum_required(VERSION 3.2)
 
+# Build using the C++ standard version 11.
+set(CMAKE_CXX_STANDARD 11)

```

Comment:
This should be set with `target_compile_features`.
Ideally the aws sdk package would set the correct defaults (c++11) as a public propery and any code linking to it would automatically require a minimum of C++11

Reply:
This works.

```
target_compile_features(my-example PUBLIC cxx_std_11)
```

Opened https://github.com/aws/aws-sdk-cpp/issues/1338